backend/src/test/java/org/example/backend/model/FriendRequestTest.java

```diff
@@ -0,0 +1,68 @@
+package org.example.backend.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+class FriendRequestTest {
```

Comment:
You shouldn't need to test those, if you add a lombok.config to your backend/src folder.

Reply:
those = your models.

Reply:
oh, good to know 🥳
I was already trying to find a solution, to not manually test them. Didn't really think, that everytime I use the lombok methods, I also have to test them 🤔
Thanks for the tip 😊